src/Authorization.AspNetCore/AuthorizationValidationRule.cs

```diff
@@ -68,108 +71,120 @@ public Task<INodeVisitor> ValidateAsync(ValidationContext context)
             }));
         }
 
-        private async Task AuthorizeAsync(
-            INode node,
-            IProvideMetadata type,
-            ValidationContext context,
-            OperationType operationType)
+        private async Task AuthorizeAsync(INode node, IProvideMetadata type, ValidationContext context, OperationType? operationType)
         {
-            if (type == null || !type.RequiresAuthorization())
```

Comment:
`RequiresAuthorization` here is redundant since it just calls `GetPolicies` nad check result.